src/rdb/datum.php

```diff
@@ -67,6 +67,9 @@ function nativeToDatum($v) {
         return new StringDatum($v);
     } else if (is_object($v) && is_subclass_of($v, "\\r\\Query")) {
         return $v;
+    } else if (is_a($v, 'DateTime')) {
```

Comment:
One more thing: Can you make this check for `is_subclass_of($v, "DateTimeInterface")` instead?
That way it would also handle objects of type DateTimeImmutable http://php.net/manual/en/class.datetimeimmutable.php